### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by jlovejoy**
remove request license instructions as discussed in #1211 
update bi-weekly call description

### CODE REVIEW

**Comment by jlovejoy** [comment]
need help with the how-to-get-started info. need better explanation re: XML markup in particular - where can we point people (this probably points to the need for better documentation around that... )

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,42 +1,36 @@
...
-* **this Github repo**: comments and issues and PRs related to specific changes to the files that comprise the SPDX License List, e.g., update a URL, recommend additional markup for matching purposes, or other such changes. Requests for new license or exceptions are tracked here, see more info on that process below.
+* **join the mailing list**: Please introduce yourself and let us know a bit about your interest in SPDX! The mailing list is our traditional form of communication. Join the mailing list, see archive, and manage your subscription at [via lists.spdx.org](https://lists.spdx.org/g/Spdx-legal).
+* **join the bi-weekly calls**: Bi-weekly calls and conference call info is sent prior to the calls via the mailing list. If you join the mailing list, you should get a recurring invite at the beinning of each calendar year. Meeting minutes for the calls are in the [SPDX meetings repo](https://github.com/spdx/meetings/tree/master/legal); older meeting minutes can be found at http://wiki.spdx.org/
```
Comment (mkdolan) [inline CONTRIBUTING.md:15]:
```suggestion
* **join the bi-weekly calls**: Bi-weekly conference call info is sent prior to the calls via the mailing list. If you join the mailing list, you should get a recurring invite at the beginning of each calendar year. Meeting minutes for the calls are in the [SPDX meetings repo](https://github.com/spdx/meetings/tree/master/legal); historical meeting minutes can be found at http://wiki.spdx.org/
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,42 +1,36 @@
...
+* Help prepare files for licenses approved to be added to the SPDX License List by following the [new license workflow](https://github.com/spdx/license-list-XML/blob/master/DOCS/new-license-workflow.md)
 
-3. Submit your request via the SPDX Online Tool [Submit New License](https://tools.spdx.org/app/submit_new_license/) using the guidance provided there. Also see the [explanation of the fields contained on the list](DOCS/license-fields.md).
-   - Note: You must have a Github account in order to use this tool/process.
-   - Note: Whether you use the [Submit New License](https://tools.spdx.org/app/submit_new_license/) tool or submit manually via a new issue, you will need to provide *all* of the information as per the form in the tool. Incomplete submissions will delay the process.
+* Review PRs 
+
+* Request a new license be added to the SPDX License List by following the instructions HERE ADDLINK
```
Comment (mkdolan) [inline CONTRIBUTING.md:28]:
```suggestion
* Request a new license be added to the SPDX License List by following the instructions in the [CONTRIBUTING](https://github.com/spdx/license-list-XML/blob/master/CONTRIBUTING.md) file.
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,42 +1,36 @@
...
+
+* Email the mailing list about how you are using or questions about the SPDX License List 
+*
```
Comment (mkdolan) [inline CONTRIBUTING.md:31]:
```suggestion
```

**Review by mkdolan — CHANGES_REQUESTED** [review]
Looks good to me. I suggested a few cleanups.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -1,42 +1,36 @@
...
-* **join the mailing list**: Please introduce yourself and let us know a bit about your interest in SPDX! The mailing list is our traditional form of communication. Join the mailing list, see archive, and manage your subscription [via lists.spdx.org](https://lists.spdx.org/g/Spdx-legal).
-* **bi-weekly calls**: used to discuss topics and issues that may be difficult to only discuss via email or via Github. Information on the bi-weekly calls and the working area for the SPDX Legal Team, can be found on the [SPDX legal team wiki](https://wiki.spdx.org/view/Legal_Team)
-* **this Github repo**: comments and issues and PRs related to specific changes to the files that comprise the SPDX License List, e.g., update a URL, recommend additional markup for matching purposes, or other such changes. Requests for new license or exceptions are tracked here, see more info on that process below.
+* **join the mailing list**: Please introduce yourself and let us know a bit about your interest in SPDX! The mailing list is our traditional form of communication. Join the mailing list, see archive, and manage your subscription at [via lists.spdx.org](https://lists.spdx.org/g/Spdx-legal).
```
Comment (bsdimp) [inline CONTRIBUTING.md:14]:
```
-* **join the mailing list**: Please introduce yourself and let us know a bit about your interest in SPDX! The mailing list is our traditional form of communication. Join the mailing list, see archive, and manage your subscription at [via lists.spdx.org](https://lists.spdx.org/g/Spdx-legal).
+* **join the mailing list**: Please introduce yourself and let us know a bit about your interest in SPDX! The mailing list is our traditional form of communication. Join the mailing list, see archive, and manage your subscription [via lists.spdx.org](https://lists.spdx.org/g/Spdx-legal).
```
the old text would render as "at via lists.spdx.org." making the 'at' reduntant.

**Comment by bsdimp** [comment]
Modulo nits, this looks good to me...

File: CONTRIBUTING.md (hunk 2 above)
Comment (jlovejoy) [reply CONTRIBUTING.md:28]: I left ADDLINK b/c the request new license info is moving off the Contributing doc and to it's own doc

File: CONTRIBUTING.md (hunk 4 above)
Comment (jlovejoy) [reply CONTRIBUTING.md]: good catch, got it!

File: CONTRIBUTING.md (hunk 2 above)
Comment (mkdolan) [reply CONTRIBUTING.md:28]: Makes sense. +1

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -1,42 +1,33 @@
...
-3. Submit your request via the SPDX Online Tool [Submit New License](https://tools.spdx.org/app/submit_new_license/) using the guidance provided there. Also see the [explanation of the fields contained on the list](DOCS/license-fields.md).
-   - Note: You must have a Github account in order to use this tool/process.
-   - Note: Whether you use the [Submit New License](https://tools.spdx.org/app/submit_new_license/) tool or submit manually via a new issue, you will need to provide *all* of the information as per the form in the tool. Incomplete submissions will delay the process.
+* Review PRs 
 
-4. Submitters are expected to follow and participate in the discussion and answer any request for additional information via the issue and comments. Please note, if submitters are unresponsive for several months, the issue will be closed without making a final decision.
+* Request a new license be added to the SPDX License List by following the instructions HERE ADDLINK
 
-## Review Process
+* Email the mailing list about how you are using or questions about the SPDX License List
```
Comment (silverhook) [inline CONTRIBUTING.md:30]:
```suggestion
* Email the [mailing list](https://lists.spdx.org/g/spdx-legal) about how you are using or questions about the SPDX License List 
```

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -1,42 +1,33 @@
...
+* **join the bi-weekly calls**: Bi-weekly conference call info is sent prior to the calls via the mailing list. If you join the mailing list, you should get a recurring invite at the beginning of each calendar year. Meeting minutes for the calls are in the [SPDX meetings repo](https://github.com/spdx/meetings/tree/master/legal); historical meeting minutes can be found at http://wiki.spdx.org/
 
-Please do not raise general questions via issues in this Github repository, but use the appropriate workgroup mailing list.
+# Getting started 
+Below are some ways you can get started participating and contributing!
 
-## Request a new license or exception be added to the SPDX License List
-The SPDX Legal Team appreciates proposals for new free and open source licenses or exceptions to be added to the SPDX License List.  To request a license or exception be added to the SPDX License List, please:
+* Commenting on new license requests: Once you have familiarized yourself with the [License inclusion principles](DOCS/license-inclusion-principles.md), comment on new license requests as to whether you think the license should be included on the SPDX License List
```
Comment (silverhook) [inline CONTRIBUTING.md:20]: Where? On mailing list and/or GitHub?

Comment (jlovejoy) [reply CONTRIBUTING.md]: good catch, added a link to issues with new license/exception label

**Comment by jlovejoy** [comment]
@swinslow - I think this is ready to merge once the checks have passed

**Pull request merged**